transformer_engine/common/hadamard_transform/group_hadamard_transform_cast_fusion.cu

```diff
@@ -1011,13 +1027,21 @@ void group_hadamard_transform_cast_fusion_columnwise(
     k_tile_size = 512;
   }
 
+  // TODO: haven't decided whether to expose this as a API option or not
+  // use fast math if there is a ENV var NVTE_RHT_CAST_FUSION_USE_FAST_MATH, default to false
+  static const bool use_fast_math =
+      transformer_engine::getenv<bool>("NVTE_RHT_CAST_FUSION_USE_FAST_MATH", false);
```

Comment:
Comparing approaches:
- Check envvar (current): simple to implement, extremely obscure for users
- Add fast math to quantization config: vaguely similar to stochastic rounding (only affect NVFP4 quantization, no effect on dequantization), could be applied to other places we quantize to NVFP4, somewhat annoying to implement